### PR TITLE
💄 compress login/logout messages even more

### PR DESCRIPTION
**Pull request by Hayleia**
Changes this
📥 redgl0w, 📤 Lil_Nas_X3009, 📤 noelnadal Timeout, 📤 Ti64CLi++ Timeout, 📤 Shadow17 Timeout, 📤 ascpial Timeout, 📥 ascpial, 📤 ascpial, 📥 ascpial, 📤 ascpial Timeout, 📥 ascpial, 📤 ascpial Timeout, 📥 ascpial, 📥 233, 📤 ascpial, 📥 ascpial, 📤 ascpial Timeout, 📥 ascpial, 📤 ascpial Timeout, 📥 ascpial

into this
redgl0w 📥, Lil_Nas_X3009 📤, noelnadal ⏰, Ti64CLi++ ⏰, Shadow17 ⏰, ascpial ⏰📥📤📥⏰📥⏰📥, 233 📥, ascpial 📤📥⏰📥⏰📥

Basically, 
- use alarm clock instead of outbox tray + Timeout for a timeout
- don't repeat username if it's the same

Warning, emojis are on the right side of the username in the new code, not the left side. If it is a problem, I can probably change that.

### CODE REVIEW

**Comment by Hayleia** [comment]
Notice that the version with the emojis on the left is already done on morecompressedlogins2.
There is no work to be done, just a choice to be made.

**Comment by Hayleia** [comment]
Chose the version with emojis on the left because
- the last emoji and the username are right next to each other, quick access to information
- this was the way things were before that pull request, no need to get used to new things
